src/CompanyService.Data/Interfaces/IDepartmentRepository.cs

```diff
@@ -1,49 +1,49 @@
-﻿using LT.DigitalOffice.CompanyService.Models.Db;
+﻿using System;
+using System.Collections.Generic;
+using LT.DigitalOffice.CompanyService.Models.Db;
 using LT.DigitalOffice.CompanyService.Models.Dto.Requests.Filters;
 using LT.DigitalOffice.Kernel.Attributes;
 using Microsoft.AspNetCore.JsonPatch;
-using System;
-using System.Collections.Generic;
 
 namespace LT.DigitalOffice.CompanyService.Data.Interfaces
 {
+  /// <summary>
+  /// Represents the repository pattern.
+  /// Provides methods for working with the database of CompanyService.
+  /// </summary>
+  [AutoInject]
+  public interface IDepartmentRepository
+  {
     /// <summary>
-    /// Represents the repository pattern.
-    /// Provides methods for working with the database of CompanyService.
+    /// Adds new department to the database. Returns its Id.
     /// </summary>
-    [AutoInject]
-    public interface IDepartmentRepository
-    {
-        /// <summary>
-        /// Adds new department to the database. Returns its Id.
-        /// </summary>
-        /// <param name="department">Department to add.</param>
-        /// <returns>New department Id.</returns>
-        Guid Create(DbDepartment department);
+    /// <param name="department">Department to add.</param>
+    /// <returns>New department Id.</returns>
+    Guid Create(DbDepartment department);
 
-        /// <summary>
-        /// Get <see cref="DbDepartment"/>.
-        /// </summary>
-        DbDepartment Get(Guid? departmentId, Guid? userId);
+    /// <summary>
+    /// Get <see cref="DbDepartment"/>.
+    /// </summary>
+    DbDepartment Get(Guid? departmentId, Guid? userId);
 
-        DbDepartment Get(GetDepartmentFilter filter);
+    List<DbDepartment> Get(List<Guid> departmentsIds, bool includeUsers = false);
 
-        List<Guid> AreDepartmentsExist(List<Guid> departmentIds);
+    DbDepartment Get(GetDepartmentFilter filter);
 
-        /// <summary>
-        /// Find departments in database.
-        /// </summary>
-        /// <returns>Found departments.</returns>
-        List<DbDepartment> Find(int skipCount, int takeCount, bool includeDeactivated, out int totalCount);
+    List<Guid> AreDepartmentsExist(List<Guid> departmentIds);
```

Comment:
bad naming of method. see are - should return bool

Reply:
it is not mine

Reply:
and logic is to return all existing